src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,32 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        myResource.write(LocalDateTime.now().toString());
+        req.getRequestDispatcher("/WEB-INF/views/written.jsp").forward(req, resp);
+    }
+
+    @Override
+    public void destroy() {
+        myResource.close();
+        super.destroy();
+    }
+
+    @Override
+    public void init() throws ServletException {
```

Comment:
Better put this method before doGet method

Reply:
Done